Update/haji_002.txt

```diff
@@ -49,313 +49,313 @@ void main()
 	ClearMessage();
 
 	DrawBustshot( 3, "sprite/normal/sa1a_def_a1_0", 160, 0, 0, FALSE, 0, 0, 0, 0, 0, 0, 0, 20, 200, TRUE );
 
 //沙都子rvS23/04/VTH_satoko1005.「私も大丈夫ですわ」
 	PlaySE(4, "ps3/s23/04/vth_satoko1005", 256, 64);
 	OutputLine(NULL, "「私も大丈夫ですわ」",
-			NULL, "", Line_Normal);
+			NULL, "I'm fine as well.", Line_Normal);
 	ClearMessage();
 
 	DrawBustshot( 5, "sprite/normal/ri1_warai_a1_0", -160, 0, 0, FALSE, 0, 0, 0, 0, 0, 0, 0, 18, 200, TRUE );
 
 //梨花rvS23/05/VTH_rika1006.「もちろんなのですよ、にぱ～☆」
 	PlaySE(4, "ps3/s23/05/vth_rika1006", 256, 64);
 	OutputLine(NULL, "「もちろんなのですよ、にぱ～☆」",
-			NULL, "", Line_Normal);
+			NULL, "Of course I'm free, nipah～☆", Line_Normal);
 	ClearMessage();
 
 //r不都合のある人間はいないようだった。
 	OutputLine(NULL, "不都合のある人間はいないようだった。",
-			NULL, "", Line_Normal);
+			NULL, "Looks like everyone can make it.", Line_Normal);
 	ClearMessage();
 
 //圭一rvS23/01/VTH_k11008.「お、お！　ってことは何だ！kvS23/01/VTH_k11009.　今度の日曜日はプールってわけか？！kvS23/01/VTH_k11010.　へへ、そいつぁ楽しみだなぁ！」
 	PlaySE(4, "ps3/s23/01/vth_k11008", 256, 64);
 	OutputLine(NULL, "「お、お！　ってことは何だ！",
-			NULL, "", Line_WaitForInput);
+			NULL, "Oh, oh! What's this?", Line_WaitForInput);
 	PlaySE(4, "ps3/s23/01/vth_k11009", 256, 64);
 	OutputLine(NULL, "　今度の日曜日はプールってわけか？！",
-			NULL, "", Line_WaitForInput);
+			NULL, "Next Sunday we're going to the pool?", Line_WaitForInput);
 	PlaySE(4, "ps3/s23/01/vth_k11010", 256, 64);
 	OutputLine(NULL, "　へへ、そいつぁ楽しみだなぁ！」",
-			NULL, "", Line_Normal);
+			NULL, "Hehe, that sounds like fun.", Line_Normal);
 	ClearMessage();
 
 	DrawBustshot( 3, "sprite/normal/sa1a_warai_a1_0", 160, 0, 0, FALSE, 0, 0, 0, 0, 0, 0, 0, 20, 200, TRUE );
 
 //沙都子rvS23/04/VTH_satoko1006.「をっほっほっほ！　楽しみですわねぇ！kvS23/04/VTH_satoko1007.　綺麗になった市民プール、今から本当に楽しみですわー！」
 	PlaySE(4, "ps3/s23/04/vth_satoko1006", 256, 64);
 	OutputLine(NULL, "「をっほっほっほ！　楽しみですわねぇ！",
-			NULL, "", Line_WaitForInput);
+			NULL, "Ohohoho! It'll be fun.", Line_WaitForInput);
 	PlaySE(4, "ps3/s23/04/vth_satoko1007", 256, 64);
 	OutputLine(NULL, "　綺麗になった市民プール、今から本当に楽しみですわー！」",
-			NULL, "", Line_Normal);
+			NULL, "I'll be looking forward to the new pool all week!", Line_Normal);
 	ClearMessage();
 
 //梨花rvS23/05/VTH_rika1007.「……みんなでばちゃばちゃお水遊び、楽しみなのです」
 	PlaySE(4, "ps3/s23/05/vth_rika1007", 256, 64);
 	OutputLine(NULL, "「……みんなでばちゃばちゃお水遊び、楽しみなのです」",
-			NULL, "", Line_Normal);
+			NULL, "I can't wait to splish splash with everyone.", Line_Normal);
 	ClearMessage();
 
 //圭一rvS23/01/VTH_k11011.「へへ、単なるお水遊びなら楽しみなんだけどな。kvS23/01/VTH_k11012.どうせ当日はプールで何かとんでもない部活とかがあるんだろ？」
 	PlaySE(4, "ps3/s23/01/vth_k11011", 256, 64);
 	OutputLine(NULL, "「へへ、単なるお水遊びなら楽しみなんだけどな。",
-			NULL, "", Line_WaitForInput);
+			NULL, "It'd be fine if it was just splish splashing though.", Line_WaitForInput);
 	PlaySE(4, "ps3/s23/01/vth_k11012", 256, 64);
 	OutputLine(NULL, "どうせ当日はプールで何かとんでもない部活とかがあるんだろ？」",
-			NULL, "", Line_Normal);
+			NULL, "There's going to be some insane club activty at the pool, right?", Line_Normal);
```

Comment:
I feel like this sentence doesn't lead into the next very well (I think the two together should be a "it would be great if it was just this, but I know that's not the case" kind of thing which requires one of them to have some sort of "but" or "however")